bears/general/SpaceConsistencyBear.py

```diff
@@ -89,6 +112,7 @@ def run(self,
                     diffs={filename: diff},
                     file=filename,
                     line=line_number,
+                    aspect=Formatting('Python'),
```

Comment:
This is not the intended way of adding an aspect to a `Result`. Results should reference the same aspect instances from `self.section.aspects` that were used for performing the analyses leading to the respective results

We might need some little redesign here...

Reply:
And accordingly, the statement ``'Line contains following spacing inconsistencies:' + inconsistencies`` can be molded for that particular aspect like ``'Line contains following indentation ( the aspect asked by the user) inconsistencies:' + inconsistencies``. So, the result should be shown for only those aspects which are asked by the user.